libs/librrgraph/src/base/rr_graph_utils.cpp

```diff
@@ -139,7 +139,7 @@ vtr::vector<RRNodeId, std::vector<RREdgeId>> get_fan_in_list(const RRGraphView&
 
     //Walk the graph and increment fanin on all dwnstream nodes
     rr_graph.rr_nodes().for_each_edge(
-        [&](RREdgeId edge, __attribute__((unused)) RRNodeId src, RRNodeId sink) {
+        [&](RREdgeId edge, RRNodeId src, RRNodeId sink) -> void {
```

Comment:
I agree that `__attribute__((unused))` should not be used since it only works for some compilers; however `src` is still unused. We can just void cast it in the next line.
```
(void)src;
```

Reply:
Done! thank you